client/my-sites/plans/jetpack-plans/jetpack-complete-page/components/cta-buttons/style.scss

```diff
@@ -0,0 +1,40 @@
+.jetpack-complete-page__cta-buttons {
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
```

Comment:
Design mock-ups have these buttons as left-aligned, so we can safely remove `justify-content: center;`.